apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -41,36 +89,52 @@ class WebLabTest extends Component {
       );
     });
 
+  verifyBrambleMountable = () => {
+    // Here we just want to check that WebLab reaches a MOUNTABLE state,
+    // i.e. we want a "blank" load and will not provide a project or file paths
+    document.getElementById('empty-bramble-container').src =
+      '/weblab/host?blank_load=true';
+    return this.pollBrambleStatus({timeout: 3000, interval: 500});
+  };
+
   runWebLabTest = () => {
-    this.setState({
-      [STATUS_COMPUTING_IN_THE_CORE]: Status.ATTEMPTING,
-      [STATUS_CODE_PROJECTS]: Status.ATTEMPTING,
-      renderCallToAction: false
-    });
+    this.setState(
+      {
+        [STATUS_COMPUTING_IN_THE_CORE]: Status.ATTEMPTING,
+        [STATUS_CODE_PROJECTS]: Status.ATTEMPTING,
+        [STATUS_BRAMBLE_MOUNTABLE]: Status.ATTEMPTING,
+        renderCallToAction: false
+      },
+      () => {
+        const webLabChecksComplete = Promise.all([
+          ...domainDependencies.map(this.verifyDomainAccess),
+          this.verifyBrambleMountable()
+        ]);
 
-    const webLabTestPromise = Promise.all(
-      imageAccessTests.map(this.promisifiedTestImageAccess)
+        webLabChecksComplete.then(() => {
+          this.setState({renderCallToAction: true});
+        });
+      }
     );
-
-    webLabTestPromise.then(() => {
-      this.setState({renderCallToAction: true});
-    });
   };
 
-  renderTestStatus = test => {
-    return {
-      [Status.WAITING]: 'Not complete',
-      [Status.ATTEMPTING]: 'Connecting...',
-      [Status.SUCCEEDED]: 'Success',
-      [Status.FAILED]: 'Failed'
-    }[this.state[test]];
-  };
+  renderStatusText = status => (
+    <p>{`${
+      {
+        [Status.WAITING]: 'Not complete',
+        [Status.ATTEMPTING]: 'Connecting...',
+        [Status.SUCCEEDED]: 'Success',
+        [Status.FAILED]: 'Failed'
+      }[status]
+    }`}</p>
+  );
 
   renderCallToAction = () => {
     const testFailed = [
       STATUS_CODE_PROJECTS,
-      STATUS_COMPUTING_IN_THE_CORE
-    ].some(test => this.state[test] === Status.FAILED);
+      STATUS_COMPUTING_IN_THE_CORE,
+      STATUS_BRAMBLE_MOUNTABLE
+    ].some(test => this.state[test] !== Status.SUCCEEDED);
 
     if (testFailed) {
       return (
```

Comment:
(I can't comment directly on the text below) Do we want the action required to be more specific now that we are testing bramble loading? Do we have some steps for debugging? We've been sharing this [article](https://support.code.org/hc/en-us/articles/360016804871-Troubleshooting-Web-Lab-problems) with users

Reply:
We could definitely tell the user to unblock the domains if one of the first tests fails, but I'm not sure the Bramble-related recommendations are as straightforward. For example, if the user is in incognito mode on Firefox, the Bramble test will fail, but if the Bramble test fails, we can't be sure it's because they're in Firefox incognito mode.

Is the larger question whether we want a user to be able to use the test page and the troubleshooting steps in concert from the get-go? I think I was sort of assuming that users would only get directed to this page after filing a support ticket with Hannah, and she/we would be using this to verify they had done the troubleshooting steps, but there might be smarter ways to use these tools together.

Reply:
good point. If it's more complex than we could easily write up, we can keep it as-is for now and maybe as we learn more we will figure out a better set of debugging instructions

Reply:
I could see us adding a link to this page to an error message in Web Lab or to the [IT requirements page](https://code.org/educate/it). If there's a likely cause, we could give them an action to take before reaching out to Hannah. For example:
"Bramble could not be mounted. This often happens because iframes are blocked or you are using this page in incognito mode. There may be additional error messaging in your browser's developer console. Please reach out to your IT department for help. If additional guidance is needed, contact support@code.org."

Reply:
I have permission from @jmkulwik to add this in my current/follow-up PR with the filesystem test.